packages/api/routes/routes.js

```diff
@@ -30,6 +37,17 @@ function register(app) {
   app.route("/list").get(cors, list());
 
   app.post("/autosync/forceSyncAll", cors, forceSyncAll());
-}
 
+  // API Keys
+  app
+    .route("/apikey")
+    .get(cors, getAPIKeysByUser())
+    .post(cors, createAPIKey())
+    .delete(cors, deleteAPIKey());
+
+  app
+    .route("/user")
+    .get(cors, getUserByAPIKey())
+    .delete(cors, deleteAPIKeysByUser());
```

Comment:
My initial thought process around this was that since this endpoint essentially performs a delete action around a user's identity, it made sense to have the endpoint be identified by something like `/user`. Also, I was trying to keep things as simple as possible by using fewer query parameters and having each middleware focused upon atomic use-cases. Having said that though, I agree that it can also be combined with the deleteAPIKey middleware and I'm open to making that change since it would reduce the number of separate endpoints.

Reply:
Yeah, I like your thought process.  But I do think it should be combined into the deleteAPIKey middleware.  That middleware can become a mini router too, where it just looks for a query param and then passes off to the appropriate middleware handler (if `user` pass to `deleteByUser`, if `expired` pass to `deleteByExpired`, etc).

However!  I think this is fine as is.  We can make that adaptation later.

Reply:
Oh and one more note about the URL namespacing of deleteAPIKEysByUser, I really feel that it should be under `/apikey`.  The subject of the operation is API key(s), and user is just a query parameter.  Imagine other queries by which we might want to delete API keys, such as by `expired`, by `createDate`, etc.  We would not put those under `/expired` and `/createDate`, we'd put them under `/apikey`.  I think the same is true of this.

Reply:
Agreed! Maybe we can do this post MVP. I'll create an issue for it.